content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -294,116 +294,121 @@ To form a single point of configuration for all telemetry emitted directly from
 
-When you [create a RUM application][2], confirm the `env` and `service` names.
+   When you [create a RUM application][2], confirm the `env` and `service` names.
 
 
 [1]: /real_user_monitoring/connect_rum_and_traces/
 [2]: /real_user_monitoring/browser/#setup
-{{% /tab %}}
+   {{% /tab %}}
 
-{{% tab "Custom Metrics" %}}
+   {{% tab "Synthetics" %}}
 
-Tags are added in an append-only fashion for [custom statsd metrics][1]. For example, if you have two different values for `env`, the metrics are tagged with both environments. There is no order in which one tag overrides another of the same name.
+   If you're using [connected Synthetic browser tests and traces][1], specify a URL to send headers to under the **APM Integration for Browser Tests** section of the [Integration Settings page][2]. 
 
-If your service has access to `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`, then the DogStatsD client automatically adds the corresponding tags to your custom metrics.
+   You can use `*` for wildcards, for example: `https://*.datadoghq.com`.
 
-**Note**: The Datadog DogStatsD clients for .NET and PHP do not yet support this functionality.
+[1]: /synthetics/apm/
+[2]: https://app.datadoghq.com/synthetics/settings/integrations
+   {{% /tab %}}
 
-[1]: /metrics/
-{{% /tab %}}
+   {{% tab "Custom Metrics" %}}
 
-{{% tab "System Metrics" %}}
+   Tags are added in an append-only fashion for [custom statsd metrics][1]. For example, if you have two different values for `env`, the metrics are tagged with both environments. There is no order in which one tag overrides another of the same name.
 
-`env` and `service` can also be added to your infrastructure metrics.
+   If your service has access to `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`, then the DogStatsD client automatically adds the corresponding tags to your custom metrics.
 
-The tagging configuration for service metrics lives closer to the Agent in non-containerized contexts.
-Given that this configuration does not change for each invocation of a service's process, adding `version`
-to the configuration is not recommended.
+   **Note**: The Datadog DogStatsD clients for .NET and PHP do not yet support this functionality.
 
-##### Single service per host
+[1]: /metrics/
+   {{% /tab %}}
 
-Set the following configuration in the Agent's [main configuration file][1]:
+   {{% tab "System Metrics" %}}
 
-```yaml
-env: <ENV>
-tags:
-    - service:<SERVICE>
-```
+   You can add `env` and `service` tags to your infrastructure metrics.
 
-This setup guarantees consistent tagging of `env` and `service` for all data emitted by the Agent.
+   The tagging configuration for service metrics lives closer to the Agent in non-containerized contexts.
+   Given that this configuration does not change for each invocation of a service's process, adding `version`
+   to the configuration is not recommended.
 
-##### Multiple services per host
+   #### Single service per host
```

Comment:
this whole section seems to be weirdly indented. not sure if it makes a difference but should probably be dedented for consistency